modules/services/OsmService.js

```diff
@@ -651,6 +651,55 @@ export class OsmService extends AbstractSystem {
   }
 
 
+  /**
+   * _parseUserPreferencesXML
+   * @param xml
+   * @param callback
+   */
+  _parseUserPreferencesXML(xml, callback) {
```

Comment:
This is pretty cool actually - We've talked about reading/storing user preferences from the OSM API for a while:  https://github.com/openstreetmap/iD/issues/3002

We can expand this later and use it for other things.